packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -84,10 +88,47 @@ export interface ITreeConfigurationOptions {
 	 * bit slower.
 	 */
 	enableSchemaValidation?: boolean;
+
+	/**
+	 * If `true`, constructing the configuration will error if any valid input data could parse ambiguously
```

Comment:
I think that this "summary" section could be a little less precise/technical in favor of being more high level and readable. For example:

"A flag which ensures that the schema can't allow for any ambiguity when constructing nodes.
If enabled, and the schema does allow for such ambiguity, an error will be thrown during initialization."

After reading that summary, an unfamiliar reader still won't know exactly what this flag does, but they'll know enough to know whether or not they care and want to keep reading. I think that's appropriate for a summary section.

Reply:
done.